frontend/appflowy_flutter/lib/plugins/document/presentation/share/share_button.dart

```diff
@@ -52,27 +49,30 @@ class DocumentShareButton extends StatelessWidget {
     );
   }
 
-  void _handleExportData(ExportDataPB exportData) {
+  void _handleExportData(BuildContext context, ExportDataPB exportData) {
     switch (exportData.exportType) {
```

Comment:
```suggestion
  void _handleExportData(BuildContext context, ExportDataPB exportData) {
    String message = switch (exportData.exportType) {
        ExportType.Markdown => LocaleKeys.settings_files_exportFileSuccess.tr(),
        ExportType.Link => LocaleKeys.settings_link_copiedToClipboardSuccess.tr(),
        ExportType.text => LocaleKeys.settings_text_copiedToClipboardSuccess.tr(),
      };
      
      showSnakBarMessage(context, message);
```

Reply:
The other cases have not been implemented yet, so there is no need to handle them at the moment.